packages/back-end/src/controllers/features.ts

```diff
@@ -735,12 +734,17 @@ export async function getFeatureById(
       env.rules?.forEach((r) => {
         if (r.type === "experiment") {
           expIds.add(r.trackingKey || feature.id);
+          if (r.values.length) {
```

Comment:
@jdorn What do you think about this approach to generating a variation id (if one does not exist) for a feature variation? I'm doing this in several other spots as well to ensure that anywhere we're trying to get a feature/experiment variation, if an id doesn't exist, we're generating one.

Reply:
This is probably better handled by a one-time data migration that is run after the feature is deployed. Do we currently do migrations in Mongo?